File: loianeTraining/javaBasico/exercicios-aulas-18-19/Ex3.java
public class Ex3 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 };
        int[] b = new int[15];

        for (int i = 0; i < a.length; i++) {
            b[i] = a[i] * a[i];
            System.out.println("A " + a[i] + "\t" + "B " + b[i] + "\n");
        }
    }
}

File: Loiane_Training_C#/Basico/D/Ex3.cs
using System;

class Ex3
{
    static void Main(string[] args)
    {
        bool repeat = true;
        while (repeat)
        {
            Console.Write("Digite seu nome: ");
            string name = Console.ReadLine();

            Console.Write("Digite sua idade: ");
            int age = int.Parse(Console.ReadLine());

            Console.Write("Digite seu salario : ");
            double salary = double.Parse(Console.ReadLine());

            Console.Write("Digite seu sexo: ");
            char gender = char.Parse(Console.ReadLine());

            Console.Write("Digite seu estado civil: ");
            char state = char.Parse(Console.ReadLine());

            if (name.Length > 3 &&
                (age > 0 && age < 150) &&
                salary > 0 &&
                (gender == 'f' || gender == 'm') &&
                (state == 's' || state == 'c' || state == 'v' || state == 'd'))
            {
                repeat = false;
            }
        }
    }
}